ana_scans.py

```diff
@@ -123,9 +124,9 @@ def launchAnaArgs(anaType, cName, cType, scandate, scandatetrim=None, ztrim=4.0,
   #Execute Commands
   try:
     log = file("%s/anaLog.log"%(dirPath),"w")
-   
-    runCommand(cmd,log)
-    #runCommand(cmd)
+ 
+    #runCommand(cmd,log)
+    runCommand(cmd)
     for key in postCmds:
       runCommand(postCmds[key])
```

Comment:
Yes I see what you mean, misunderstanding from my side.  I thought you were asking for it to be one long list of strings where each command is just the next entry in the list rather than a list of lists where each entry is a list specifying the command.

In this case yes it could be this way and I agree it will be cleaner rather than having to continually increase the key idx.  I will update.